includes/wcs-resubscribe-functions.php

```diff
@@ -21,20 +21,8 @@
  * @since 1.0.0 - Migrated from WooCommerce Subscriptions v2.0
  */
 function wcs_order_contains_resubscribe( $order ) {
-
-	if ( ! is_a( $order, 'WC_Abstract_Order' ) ) {
-		$order = wc_get_order( $order );
-	}
-
-	$related_subscriptions = wcs_get_subscriptions_for_resubscribe_order( $order );
-
-	if ( wcs_is_order( $order ) && ! empty( $related_subscriptions ) ) {
-		$is_resubscribe_order = true;
-	} else {
-		$is_resubscribe_order = false;
-	}
-
-	return apply_filters( 'woocommerce_subscriptions_is_resubscribe_order', $is_resubscribe_order, $order );
+	$related_subscription_ids = wcs_get_subscription_ids_for_order( $order, 'resubscribe' );
+	return apply_filters( 'woocommerce_subscriptions_is_resubscribe_order', ! empty( $related_subscription_ids ), $order );
```

Comment:
This applies to all the `contains_x()` functions but these filters use to always receive an order object. Given these functions can accept an ID, they will now receive whatever was passed in -- ID or object. 

Unfortunately I think this means we will need to return to the 

```php 
if ( ! is_a( $order, 'WC_Abstract_Order' ) ) {
    $order = wc_get_order( $order );
}
```

Reply:
Good catch @james-allan!

Reply:
Just pushed up d4927961 to address this! Going to run through some more tests on these latest changes